presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestJoinQueries.java

```diff
@@ -2459,4 +2516,13 @@ protected Session noJoinReordering()
                 .setSystemProperty(JOIN_DISTRIBUTION_TYPE, FeaturesConfig.JoinDistributionType.PARTITIONED.name())
                 .build();
     }
+
+    private Long getTableRowCount(String tableName)
```

Comment:
Would it make sense for return value to be 'long' to eliminate the possibility of null?